meta-mender-core/recipes-mender/mender-client/mender-client_3.0.1.bb

```diff
@@ -0,0 +1,31 @@
+require mender-client.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=3.0.x"
+
+# Tag: 3.0.1
+SRCREV = "90da1266710ba89f061e511d3e2cb46a7258088f"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
```

Comment:
In `dunfell`, the `3.x` series of recipes are down-prioritized because we cannot introduce a major bump in mender-client version on an stable branch. Documented [here](https://docs.mender.io/overview/compatibility#mender-client-and-yocto-project-version).